src/app/core/signing.service.ts

```diff
@@ -90,7 +90,7 @@ export class SigningService {
     // timeout so that it sets the selection range AFTER the textbox is modified
     setTimeout(() => {
       textbox.focus();
-      textbox.scrollTop = scrollPos;
+      textbox.scrollTop = scroll_pos;
```

Comment:
Improper variable naming
```suggestion
      textbox.scrollTop = scrollPos;
```